pkg/handlers/rendered_contents.go

```diff
@@ -94,9 +95,16 @@ func (h *Handler) GetAppRenderedContents(w http.ResponseWriter, r *http.Request)
 		return
 	}
 
-	_, chartsFilesMap, err := kustomize.GetRenderedChartsArchive(archivePath, d.Name, kotsKinds.GetKustomizeBinaryPath())
+	_, v1Beta1ChartsFilesMap, err := kustomize.GetRenderedChartsArchive(archivePath, d.Name, kotsKinds.GetKustomizeBinaryPath())
```

Comment:
The name `GetRenderedChartsArchive` doesn't distinguish between v1beta1 and v1beta2, how about renaming it to:
```suggestion
	_, v1Beta1ChartsFilesMap, err := kustomize.GetRenderedV1Beta1ChartsArchive(archivePath, d.Name, kotsKinds.GetKustomizeBinaryPath())
```